src/components/header/components/desktop/Desktop.component.tsx

```diff
@@ -7,67 +7,58 @@ import { LightIconComponent } from '@/components/header/components/shared/icons/
-                    <li>
-                        <button className={styles.theme}>
-                            <LightIconComponent customClasses={styles.offset_sm} />
-                        </button>
-                    </li>
-                    <li>
-                        <DividerIconComponent customClasses={styles.offset_sm} />
-                    </li>
-                    <li>
-                        <button className={styles.theme}>
-                            <DarkIconComponent customClasses={styles.offset_sm} />
-                        </button>
-                    </li>
-                </ul>
-            </div>
-            <div className={wrapperClasses}>
-                <ul className={styles.list}>
-                    <li>
-                        <a className={linkClasses} href="/about">
-                            About
-                        </a>
-                    </li>
-                    <li>
-                        <a className={linkClasses} href="/products">
-                            Products
-                        </a>
-                    </li>
-                    <li>
-                        <CartComponent />
-                    </li>
-                    <li className={styles.offset_md}>
-                        <a className={styles.login} href="/login">
-                            <LoginIconComponent customClasses={styles.offset_sm} />
-                            Login
-                        </a>
-                    </li>
-                    <li>
-                        <a className={styles.sign_up} href="/sign-up">
-                            <SignUpIconComponent customClasses={styles.offset_sm} />
-                            Sign up
-                        </a>
-                    </li>
-                </ul>
-            </div>
-        </header>
-    );
-};
+export const DesktopComponent = ({ setSelectedNawBarItem }: NavBarItem) => (
+    <header className={styles.header}>
+        <div className={`${styles.flex} ${styles.align_center}`}>
+            <LogoComponent />
+            <ul className={styles.theme}>
+                <li>
+                    <button className={styles.theme}>
+                        <LightIconComponent customClasses={styles.offset_sm} />
+                    </button>
+                </li>
+                <li>
+                    <DividerIconComponent customClasses={styles.offset_sm} />
+                </li>
+                <li>
+                    <button className={styles.theme}>
+                        <DarkIconComponent customClasses={styles.offset_sm} />
+                    </button>
+                </li>
+            </ul>
+        </div>
+        <div className={`${styles.flex} ${styles.align_center}`}>
+            <ul className={styles.list}>
+                <li>
+                    <button className={`${styles.text_decoration} ${styles.color_white}`} onClick={() => setSelectedNawBarItem('About')}>
```

Comment:
![Screenshot 2024-05-08 at 18 04 43](https://github.com/3apy6a/fe-react-2024/assets/115080771/e0660e07-f5c4-4112-a6bd-5524a2163730)

you need to remove border when hovering over the button

Reply:
done